src/views/transactions/deposit/components/deposit-form/deposit-form.view.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import { TxType } from "@hermeznetwork/hermezjs/src/enums";
+import { BigNumber } from "@ethersproject/bignumber";
+
+import useDepositFormStyles from "src/views/transactions/deposit/components/deposit-form/deposit-form.styles";
+import { EthereumAccount, FiatExchangeRates } from "src/domain/hermez";
+import FormContainer from "src/views/transactions/components/form-container/form.container.view";
+import { AsyncTask, isAsyncTaskCompleted } from "src/utils/types";
+import { EstimatedL1Fee } from "src/domain";
+import SelectedAccount from "src/views/transactions/components/selected-account/selected-account.view";
+import DepositAlert from "src/views/transactions/deposit/components/deposit-alert/deposit-alert.view";
+import Fee from "src/views/transactions/components/fee/fee.view";
+import { AmountInputChangeEventData } from "src/views/shared/amount-input/amount-input.view";
+import TransactionAmountInput from "src/views/transactions/components/transaction-amount-input/transaction-amount-input.view";
+import PrimaryButton from "src/views/shared/primary-button/primary-button.view";
+
+export interface TxData {
+  amount: BigNumber;
+  from: EthereumAccount;
+}
+
+interface DepositFormStateProps {
+  account: EthereumAccount;
+  estimatedDepositFeeTask: AsyncTask<EstimatedL1Fee, Error>;
+  preferredCurrency: string;
+  fiatExchangeRatesTask: AsyncTask<FiatExchangeRates, string>;
+}
+
+interface DepositFormHandlerProps {
+  onGoToChooseAccountStep: () => void;
+  onSubmit: (txData: TxData) => void;
+}
+
+type DepositFormProps = DepositFormStateProps & DepositFormHandlerProps;
+
+const DepositForm: React.FC<DepositFormProps> = ({
+  account,
+  estimatedDepositFeeTask,
+  preferredCurrency,
+  fiatExchangeRatesTask,
+  onGoToChooseAccountStep,
+  onSubmit,
+}) => {
+  const classes = useDepositFormStyles();
+  const [amount, setAmount] = React.useState(BigNumber.from(0));
+  const [isAmountValid, setIsAmountValid] = React.useState<boolean | undefined>(undefined);
+  const [showInFiat, setShowInFiat] = React.useState(false);
+  const [areFundsExceededDueToFee, setAreFundsExceededDueToFee] = React.useState(false);
+
+  function isSubmitButtonDisabled() {
+    if (isAmountValid === false || areFundsExceededDueToFee) {
+      return true;
+    }
+
+    return false;
+  }
```

Comment:
`const isSubmitButtonDisabled = isAmountValid === false || areFundsExceededDueToFee;`?

The same applies to `exit`, `force-exit`, and `transfer`

Reply:
Right! Updated all the forms

Reply:
I'm curious, why keep the function?

Reply:
Didn't see the function to const change, fixed!